### ISSUE TITLE
PG and default (0)::numeric

### ISSUE THREAD

**Issue by DrDeath72** [yiisoft/yii2#12483]
`(0)::numeric` set `'(0)'` not `0`.

`"user_timezone" numeric(5,2) DEFAULT (0)::numeric NOT NULL` set `'user_timezone' => '(0)'`

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.9

**Comment by DrDeath72** [comment]
regex working well, default not...
http://www.phpliveregex.com/p/h1h

**Comment by SilverFire** [comment]
Please, give us a little bit more information.

What are you trying to do? How can be reproduce the problem? What is the expected result?

**Comment by DrDeath72** [comment]
do `loadDefaultValues()` after create, db field and result in first post

**Comment by cebe** [comment]
please show the code you use for creation of the column and also the code to read it.

**Comment by cebe** [comment]
unable to reproduce, created a unit test to verify that it works: 06b045c

**Comment by DrDeath72** [comment]
```
class ActiveRecord extends \yii\db\ActiveRecord {

	public function init() {
		parent::init();
	    $this->loadDefaultValues();
	}
}
```
```
CREATE TABLE "public"."phpbb3_users" (
...
"user_timezone" numeric(5,2) DEFAULT (0)::numeric NOT NULL,
...
)
```
```
var_dump((new User)->attributes); die;
```
```
[
...
    'user_timezone' => '(0)'
...
]
```

**Comment by cebe** [comment]
looks like it has been fixed in 2.0.10-dev by ad796dbf225ac52b41ee6652190c1361445e3b0b

**Comment by DrDeath72** [comment]
oh... i think it's 2.0.9 code :D ok.. wait
